developer_documentation/rcdgen_markup.rst

```diff
@@ -873,9 +877,22 @@ Attributes:
 - ``logo`` - FreeRCT logo sprite.
 - ``new_game`` - New Game button sprite.
 - ``load_game`` - Load Game button sprite.
+- ``launch_editor`` - Scenario Editor button sprite.
 - ``settings`` - Settings button sprite.
 - ``quit`` - Quit button sprite.
 
+MISN
+~~~~
+Attributes:
+
+- ``internal_name`` - Internal name of the mission.
+- ``texts`` - strings_ node containing the mission's texts.
+- ``max_unlock`` - Maximum number of unlocked unsolved scenarios in the mission.
+- ``scenarios`` - Total number of scenarios in the mission.
+- ``file_N`` - Path to the FCT file for the N-th scenario.
+- ``internal_name_N`` - Internal name of the N-th scenario.
+- ``texts_N`` - strings_ node containing the N-th scenario's texts.
+
```

Comment:
Should it have author and/or version of the mission data?

I can imagine an author makes one version and then fixes or improves things, and it seems useful to be able to understand what you get. Similarly, people may copy the mission and adjust it to their liking.

I am also wondering about licenses, can an author define a license of their work?

Reply:
Since it is all wrapped in RCD files this is the same as for all custom content. Versioning can be handled via the `INFO` block (but we only have rudimentary support for that currently). 

Custom licensing is currently not possible, but if desired should also go in the `INFO`. Sometime in the far future we might want to define some contributor and publishing policies about such things. 
Over at @Widelands we decided not to let people do that, and the add-ons server simply has a confirmation checkbox that all add-ons must be published under the GPLv2.